lab-davidw/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-davidw",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "node-uuid": "^1.4.8",
+    "uuid": "^3.0.1"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^3.3.0"
+  },
+  "scripts": {
+    "test": "mocha",
+    "start": "node server",
+    "watch": "nodemon server",
+    "lint": "eslint ./.eslintrc"
```

Comment:
Good job remembering a ``lint script``, but you need to also ``npm i -D eslint`` for this to work.